src/App.js

```diff
@@ -1,19 +1,51 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
 import chatMessages from './data/messages.json';
+import ChatLog from './components/ChatLog';
+
+
+
 
 const App = () => {
+  //state
+  const [messageData, setMessageData] = useState(chatMessages);
+  let [totalLikes, setTotalLikes] = useState(0);
+
+  const updateMessageData = (updatedMessage) => {
+    const messages = messageData.map((message) => {
+      if (message.id === updatedMessage.id) {
+        return updatedMessage;
+      } else {
+        return message;
+      }
+    });
+    setMessageData(messages);
+    // console.log(messages);
+  };
+
+  const updateTotalLikes = (bool) => {
+    bool ? setTotalLikes((totalLikes += 1)) : setTotalLikes((totalLikes -= 1));
+  };
+
   return (
     <div id="App">
       <header>
-        <h1>Application title</h1>
+        <h1>Chat with {chatMessages[0].sender} and {chatMessages[1].sender}</h1>
```

Comment:
What would happen if there weren't any messages yet, or only one (consider a user has opened the chat view with a friend and is getting ready to send their first message).